test/data/Cardano/Wallet/Api/[Wallet].json

```diff
@@ -0,0 +1,2023 @@
+{
```

Comment:
75.3 KB json-file 😬 — might be a potential problem in the future, but should be fine now

Reply:
Fixed by providing cases for ´[a]´, e.g
```haskell
instance {-# OVERLAPS #-} (Checkable a, Method m)
    => JsonTestable (Verb m s ct [a]) where
```